exchanges/poloniex/poloniex.go

```diff
@@ -1167,10 +1136,13 @@ func (e *Exchange) SendAuthenticatedHTTPRequest(ctx context.Context, ep exchange
 	if requiresWrapper {
 		if val, ok := resp.(*V3ResponseWrapper); ok {
 			if val.Code != 0 && val.Code != 200 {
-				return fmt.Errorf("%w code: %d message: %s", request.ErrAuthRequestFailed, val.Code, val.Msg)
+				return fmt.Errorf("%w code: %d message: %s", request.ErrAuthRequestFailed, val.Code, val.Message)
 			}
 		}
 	}
+	if errType, ok := result.(interface{ Error() error }); ok {
+		return errType.Error()
```

Comment:
🚧 Change request:
Seeing these lines together it's obvious we need to wrap the later like the former, with `request.ErrAuthRequestFailed`